examples/Makefile

```diff
@@ -5,6 +5,15 @@ get-python:
 get-python-verify:
 	kubeless function call get-python |egrep hello.world
 
+get-python-update:
+	mkdir -p .update
+	echo "def foo():\n    return \"hello world updated\"\n" > .update/helloget.py
+	kubeless function update get-python --runtime python2.7 --handler helloget.foo --from-file .update/helloget.py
```

Comment:
I'd avoid temp files, and use `printf` instead of `echo` for deterministic "\n"-s, as:
1.
```
printf 'def foo():\n%4sreturn "hello world updated"\n' | kubeless function update get-python --runtime python2.7 --handler helloget.foo --from-file /dev/stdin
```
Or alternatively "edit" existing func impl with:
2. 
```
kubectl get configmap get-python -ogo-template='{{ index .data "helloget.py"}}'|sed 's/world/world updated/' | kubeless function update get-python --runtime python2.7 --handler helloget.foo --from-file /dev/stdin
```
I personally prefer 2. but 1. maybe clearer.

Reply:
Thanks, I prefer the first option since it is more simple and the test will work independently of the content of the updated function.